packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -366,33 +339,3 @@ async function generateTestTxForRedeemRetryable() {
   const receipt = await tx.wait()
   return receipt.transactionHash
 }
-
-function setupCypressTasks(on: Cypress.PluginEvents) {
-  let currentNetworkName: NetworkName | null = null
-  let networkSetupComplete = false
-  let walletConnectedToDapp = false
-
-  on('task', {
-    setCurrentNetworkName: (networkName: NetworkName) => {
-      currentNetworkName = networkName
-      return null
-    },
-    getCurrentNetworkName: () => {
-      return currentNetworkName
-    },
-    setNetworkSetupComplete: () => {
-      networkSetupComplete = true
-      return null
-    },
-    getNetworkSetupComplete: () => {
-      return networkSetupComplete
-    },
-    setWalletConnectedToDapp: () => {
-      walletConnectedToDapp = true
-      return null
-    },
-    getWalletConnectedToDapp: () => {
-      return walletConnectedToDapp
-    }
```

Comment:
I see a lot of helper code has been moved around in this PR - in this file + commands/ common.ts file, fundEth function and other functions too. Would it be possible to break down this PR into 2 parts - one for  updating CCTP tests and another one for cleanup?

The cleanup / refactoring can come before actually implementing cctp since it's paving the way for new cctp tests.